src/modules/aws-ebs/sub-modules/config/loadEbsConfig.test.ts

```diff
@@ -0,0 +1,84 @@
+import { loadEbsConfig } from "./loadEbsConfig";
+
+describe("loadEbsConfig()", () => {
```

Comment:
You could probably set a beforeEach to make sure that you're unsetting these variables every time.